Source/Core/InputCommon/ControllerEmu/ControlGroup/Extension.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "InputCommon/ControllerEmu/ControlGroup/Extension.h"
+
+#include <string>
+
+#include "Common/CommonTypes.h"
+#include "Core/HW/WiimoteEmu/Attachment/Attachment.h"
```

Comment:
Ah no, they aren't. They can just be replaced with an include to ControllerEmu.h to satisfy the forward decl in the header, good catch.